Gemfile

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
 
+ruby File.read('.ruby-version', mode: 'rb').chomp.prepend('~> ')
```

Comment:
Just for the record in case we want later to move this to `Gemfile.base` without enforcing the same specific version in both environments:

```ruby
ruby File.read('.ruby-version', mode: 'rb').chomp.match(/\d+\.\d+/).to_s.concat('.0').prepend('<~ ')
```